jobs/texastribune_software_engineer_05_2018.html

```diff
@@ -0,0 +1,65 @@
+---
+title: Software Engineer
+company: The Texas Tribune
+url: https://www.texastribune.org/
+location: Austin, TX
```

Comment:
You can be more specific if you want to give candidates the exact location of your office.